modules/misunderstood/src/backend/api.ts

```diff
@@ -64,10 +68,13 @@ export default async (bp: typeof sdk, db: Db) => {
     `/events/:status(${FLAGGED_MESSAGE_STATUSES.join('|')})`,
     asyncMiddleware(async (req: Request, res: Response) => {
       const { botId, status } = req.params
-      const { language } = req.query
+      const { language, start, end } = req.query
+
+      const startDate = start && unixToDate(start)
+      const endDate = end && unixToDate(end)
```

Comment:
When possible, please try to avoid duplicating code, ex here you could make an helper method extractDates for example, and call it with `const { language, startDate, endDate } = extractDates(req.query)` so it can be a one-liner in both methods